Sources/Sentry/SentrySpanContext.m

```diff
@@ -79,11 +79,13 @@ - (void)removeTagForKey:(NSString *)key
         @"type" : SentrySpanContext.type,
         @"span_id" : self.spanId.sentrySpanIdString,
         @"trace_id" : self.traceId.sentryIdString,
-        @"sampled" : self.sampled ? @"true" : @"false",
         @"tags" : _tags.copy
     }
                                                  .mutableCopy;
 
+    if (self.sampled != kSentrySampleDecisionUndecided)
+        [mutabledictionary setValue:SentrySampleDecisionNames[self.sampled] forKey:@"sampled"];
```

Comment:
```suggestion
        // Since we guard for 'undecided', we'll either send it if it's 'true' or 'false'.
        [mutabledictionary setValue:SentrySampleDecisionNames[self.sampled] forKey:@"sampled"];
```

Reply:
Well, I guess this is true in the overall project, but is a positive thinking, cause the SpanContext itself nor the serializer guard for "undecided", we are relying on any other place in the SDK to do this. But well, we are not expecting the user to go off rails in this one (hopefully).

PS.: Is just me or is weird sending the sampled value if this was sent it was sampled, missing something here?